Detectors/TPC/qc/include/TPCQC/Clusters.h

```diff
@@ -86,8 +91,10 @@ class Clusters
   CalPad mSigmaTime{"Sigma_Time"};
   CalPad mSigmaPad{"Sigma_Pad"};
   CalPad mTimeBin{"Time_Bin"};
+  CalPad mOccupancy{"Occupancy"};
   size_t mProcessedTFs{0};
   bool mIsNormalized{false};
+  float mNHBFperTF{128};
```

Comment:
The default now it `32`
```suggestion
  float mNHBFperTF{32};
```